### PR TITLE
Reduce verbose logs

### PR DESCRIPTION
**Pull request by a5anka**
Relates to #52

### CODE REVIEW

File: components/org.wso2.carbon.micro.integrator.core/src/main/java/org/wso2/carbon/micro/integrator/core/internal/ServiceComponent.java [hunk 1]
```diff
@@ -395,7 +395,7 @@ private void initializeCarbon() {
             registerCarbonServlet(httpService, defaultHttpContext);
 
 
-            log.info("Repository       : " + axis2RepoLocation);
+            log.debug("Repository       : " + axis2RepoLocation);
```
Comment (sdkottegoda) [inline components/org.wso2.carbon.micro.integrator.core/src/main/java/org/wso2/carbon/micro/integrator/core/internal/ServiceComponent.java]: We can check if debug is enabled

File: components/org.wso2.carbon.micro.integrator.core/src/main/java/org/wso2/carbon/micro/integrator/core/internal/ServiceComponent.java [hunk 2]
```diff
@@ -463,7 +463,7 @@ public void shutdown() {
         if (secMan != null) {
             secMan.checkPermission(new ManagementPermission("control"));
         }
-        log.info("Shutting down " + serverName + "...");
+        log.debug("Shutting down " + serverName + "...");
```
Comment (sdkottegoda) [inline components/org.wso2.carbon.micro.integrator.core/src/main/java/org/wso2/carbon/micro/integrator/core/internal/ServiceComponent.java]: We can check if debug is enabled

Comment (a5anka) [reply components/org.wso2.carbon.micro.integrator.core/src/main/java/org/wso2/carbon/micro/integrator/core/internal/ServiceComponent.java]: Fixed in https://github.com/wso2/micro-integrator/pull/65/commits/51edbc2cd480c5bf084bbea1edb41444a02c0b18

File: components/org.wso2.carbon.micro.integrator.core/src/main/java/org/wso2/carbon/micro/integrator/core/internal/ServiceComponent.java (hunk 1 above)
Comment (a5anka) [reply components/org.wso2.carbon.micro.integrator.core/src/main/java/org/wso2/carbon/micro/integrator/core/internal/ServiceComponent.java]: Fixed in https://github.com/wso2/micro-integrator/pull/65/commits/51edbc2cd480c5bf084bbea1edb41444a02c0b18

**Pull request merged**